csi/util.go

```diff
@@ -64,6 +65,11 @@ func parseJSONRecurringJobs(jsonRecurringJobs string) ([]longhornclient.Recurrin
 	if err != nil {
 		return nil, fmt.Errorf("invalid json format of recurringJobs: %v  %v", jsonRecurringJobs, err)
 	}
+	for _, recurringJob := range recurringJobs {
+		if _, err := cron.ParseStandard(recurringJob.Cron); err != nil {
```

Comment:
Can it handle` recurringJob.Cron == ""`?

Reply:
Yes.

ParseStandard returns a new crontab schedule representing the given standardSpec (https://en.wikipedia.org/wiki/Cron). It differs from Parse requiring to always pass 5 entries representing: minute, hour, day of month, month and day of week, in that order. It returns a descriptive error if the spec is not valid.

It accepts

- Standard crontab specs, e.g. "* * * * ?"
- Descriptors, e.g. "@midnight", "@every 1h30m"